<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: none;</string>
  </property>
  <widget class="QLabel" name="l_title_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 24pt &quot;Times New Roman Bold&quot;;
color: rgb(28, 1, 64);</string>
   </property>
   <property name="text">
    <string>EMPLOYEE MANAGMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>413</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_usernamelogin">
      <property name="styleSheet">
       <string notr="true">color: rgb(25, 1, 57);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>USERNAME:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUsername">
      <property name="styleSheet">
       <string notr="true">margin: 8px;
background-color: rgb(255, 255, 255);
padding: 6px;
border-radius: 10px;
border: 1px solid rgb(25, 1, 57);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_passwordlogin">
      <property name="styleSheet">
       <string notr="true">color: rgb(25, 1, 57);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>PASSWORD:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtPassword">
      <property name="styleSheet">
       <string notr="true">margin: 8px;
background-color: rgb(255, 255, 255);
padding: 6px;
border-radius: 10px;
border: 1px solid rgb(25, 1, 57);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 10px;
background-color: rgb(27, 1, 62);
color: rgb(255, 255, 255);



</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lCreateacc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>590</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Create an account? </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>0</y>
     <width>1061</width>
     <height>821</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../OneDrive/Desktop/templogin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSignup">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 87, 204);
border-radius: 10px;
border: none;</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_title_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(159, 205, 255);
font: 75 24pt &quot;Times New Roman&quot;;
font: 24pt &quot;Times New Roman Bold&quot;;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>l_title_3</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>btnLogin</zorder>
  <zorder>lCreateacc</zorder>
  <zorder>btnSignup</zorder>
  <zorder>l_title_2</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
